nextflow.config

```diff
@@ -9,6 +9,7 @@
 params {
 
   // Options: Mandatory
+  input = './samplesheet.csv'
```

Comment:
what does this mean? does this mean that the input has to be provided with this name?  would it conflict with the input = False

Reply:
I have removed the `input = false`, and now the nextflow.config follows the one on nf-core/nanoseq.